README.md

```diff
@@ -1,9 +1,19 @@
 This repo contains the Springer Nature front-end playbook. It details how we run software development and how we make web and mobile products together. It's filled with things we've learned based on our own experience and study of others' experiences.
 
-It is a living document that we can edit in a _public_ GitHub repo.
+See "[Changing the laws of engineering with pull requests]" for an in depth explanation of how developing a playbook like this is of benefit.
+
+This is a living document that we contribute to in a _public_ GitHub repo. Reasons for doing this in the open include (but are not limited to):
+
+1. Interacting with and learning from others. Receiving contributions from people who don't work here can help us, providing learning opportunities that we would not receive otherwise - for example, see [this contribution] from @rowanmanning.
+
+1. Providing a showcase for our work/ethics. This is _really_ useful when hiring people (for both parties). We've had very positive feedback from interviewees - it's a great recruiting tool. It also means that people are quickly up and running when they join.
+
 
 #Contributing
 
 To contribute please create a new branch for each section, then create a pull request to merge your changes in.
 
 Please keep discussion inside the issues and pull requests, avoiding Slack, hallway conversations etc. Remember that this repo is public, and the discussions we have can be of benefit to people apart from us.
+
+[this contribution]: https://github.com/springernature/frontend/pull/48#issuecomment-236139605
+[Changing the laws of engineering with pull requests]: https://github.com/springernature/frontend/pull/48#issuecomment-236139605
```

Comment:
The link of "Changing the laws of engineering with pull requests" makes me confused. It's the same as "this contribution". I think it should be linked to somewhere else?...